includes/polling/os/unix.inc.php

```diff
@@ -165,9 +165,14 @@
     $aix_descr = explode("\n", $device['sysDescr']);
     # AIX standard snmp deamon
     if ($aix_descr[1]) {
-        $serial = explode("Processor id: ", $aix_descr[1])[1];
+        # IBM Power Serial Number has 7 characters. We need last 7 characters from aixSeSerialNumber.0:
+        # aixSeSerialNumber.0 = "IBM,02061F57P"
+        $serial = substr(snmp_get($device, 'aixSeSerialNumber.0', '-Oqv', 'IBM-AIX-MIB'), -7);
```

Comment:
Assuming the serial number will always have 7 characters is a good way to run into issues.  Better to split the output or use a regex.

Reply:
Hard to believe that IBM will change their own standard. Serial Number is deeply embedded into hardware. But you made a good point. I'll change the code and split by comma.

Reply:
i am under order to work from home all this month. my LibreNMS development server does not work with company's VPN. So i can not submit changes to this PR in April.

Reply:
You can pull down this branch on any machine and make changes.